Fixing errors in CI

snbianco opened the pull request:
Updating the notebooks that are failing in our nightly CI build.

snbianco commented:
Tagging @adrianlucy @scfleming for visibility

adrianlucy commented:
Thanks Sam! Will review Thursday, but one quick minor thing that caught my eye: diff changes from `"execution_count": null` to e.g. `"execution_count": 1` suggest some cells may be executed in this commit? I think preference is for notebooks to be unexecuted in this repo, mostly (I think?) to save on space, also makes git diffs easier to read, maybe other reasons that no one ever told me.

In theory if the unwanted output in this PR takes up significant storage, I guess you might want to consider squashing the commits that take up space in the history?, but up to you / judgement call whether that's worth your time.

snbianco commented:
> Thanks Sam! Will review ~Thursday-ish, but one quick minor thing that caught my eye: diff changes from `"execution_count": null` to e.g. `"execution_count": 1` suggest some cells may be executed in this commit? I think preference is for notebooks to be unexecuted in this repo, mostly (I think?) to save on space, also makes git diffs easier to read, maybe other reasons that no one ever told me.
> 
> In theory if the unwanted output in this PR takes up significant storage, I guess you might want to consider squashing the commits that take up space in the history?, but up to you / judgement call whether that's worth your time.

That is weird, apparently the "Clear All Outputs" button in VSCode doesn't actually reset the execution count! The notebooks shouldn't actually have any output. I'll see if I can find a way to reset that variable.

adrianlucy commented:
Oh interesting, weird! At least we don't have to worry about the commit history then. FWIW in Jupyter itself I think it's "restart kernel and clear outputs"(?) but I can see it being a hassle to switch between VScode's implementation and that.

adrianlucy commented:
This is all wonderful work, thank you for fixing all this! My only significant question (and it’s a potentially naive question, not having looked too closely at what the notebook is trying to do): why not use the `TessTargetPixelFile` class in `interm_tesscut_dss_overlay.ipynb`, instead of `KeplerTargetPixelFile`? Isn’t it TESS data?

Other than that, trivial things:
- Same notebook, any reason not to delete `#data = tpf.hdulist['PIXELS'].data["FLUX"][0,:,:]` (it’s already commented-out)?
- I would delete the sentence `Kepler provides a range of software tools that should also work with TESS data.`, the link is dead (and see above about TESS vs. Kepler classes, I think this notebook was written a long time ago)
- As a mild/optional request for future PRs, for me I can usually review PRs faster if formatting fixes (like adding/removing spaces) are put in a separate commit or ideally PR from changes that alter what the code does (like removing `show_in_notebook`)

adrianlucy commented:
Oh, and still in the same notebook: the Table of Contents has a couple now non-sequiturs like "Make Gif" and possible (see abovE) "Make Kepler TPF". It should probably match the headers actually used.

snbianco commented:
Thanks for this feedback! Here's my notes:

- Regarding `Kepler` vs `TESSTargetPixelFile`: You're right, I'm not sure why I used Kepler here! Maybe I got confused since the next sentence talks about Kepler. I blame illness brain 🥲 
- Deleted the commented out line you mentioned as well as the sentence about Kepler.
- Updated the Table of Contents to match the headers.

adrianlucy reviewed (approved):
PR approved, you may merge when ready, though I would wait until Monday if I were you! Hope you can get some rest soon. (PS after merging I suggest taking a quick glance at each altered notebook once the html rending github action finishes, to ensure that changes are generally rendering as expected. It can be so finicky.)
